docs/guides/getting_started.md

```diff
@@ -47,46 +47,34 @@ Contact your babylon admin to get your **URI** service and your **userpass**.
 
 * If entries does not exist, you may need to create it (e.g for a newly deployed platform):
   We provide a tool for this, you can find it [here](https://github.com/Cosmo-Tech/backend-tf-state-to-vault)
-  This needs an initial Terraform depoyment as it uses the Terraform state to parse and populate the Vault.
-  Usefull informations can be found in the Readme of this repository !
+  This needs an initial Terraform deployment as it uses the Terraform state to parse and populate the Vault.
+  Useful information can be found in the Readme of this repository !
 
 ### Configuration
 
 At this point, you will need two variables to perform Babylon commands.
```

Comment:
j'ai supprimé two car on a 3 variables :
 At this point, you will need the following variables to perform Babylon commands.